api/types/mount/mount.go

```diff
@@ -56,3 +62,24 @@ type Driver struct {
 	Name    string            `json:",omitempty"`
 	Options map[string]string `json:",omitempty"`
 }
+
+// TmpfsOptions defines options specific to mounts of type "tmpfs".
+type TmpfsOptions struct {
+	// Size sets the size of the tmpfs, in bytes.
+	//
+	// This will be converted to an operating system specific value
+	// depending on the host. For example, on linux, it will be convered to
+	// use a 'k', 'm' or 'g' syntax. BSD, though not widely supported with
+	// docker, uses a straight byte value.
+	//
+	// Percentages are not supported.
+	SizeBytes int64
+	// Mode of the tmpfs upon creation
+	Mode os.FileMode
+
+	UID          uint32
+	GID          uint32
+	Blocks       int64
+	INodes       int64
+	MemoryPolicy string
```

Comment:
I removed it in the latest commit, but it meant `mpol` for Linux tmpfs

https://linux.die.net/man/8/mount
> mpol=[default|prefer:Node|bind:NodeList|interleave|interleave:NodeList]